src/components/app/pageEvents/components/eventsMap.tsx

```diff
@@ -52,29 +51,20 @@ export function EventsMap({ events }: { events: SWCEvents }) {
     const result: MapMarker[] = []
 
     eventsFromStateKeys.forEach(state => {
-      const coordinates = STATE_COORDS[state as keyof typeof STATE_COORDS]
-
-      for (let i = 0; i < eventsFromState[state]; i++) {
-        const offsetX = i % 2 === 0 ? 1 : -1
-        const offsetY = i % 2 === 0 ? -1 : 1
-
-        const offsetCoordinates: [number, number] =
-          i === 0
-            ? [coordinates[0], coordinates[1]]
-            : [coordinates[0] + offsetX, coordinates[1] + offsetY]
+      const coordinates = EVENT_MAP_STATE_COORDS[state as keyof typeof EVENT_MAP_STATE_COORDS]
 
-        const marker: MapMarker = {
-          id: `${state} ${i}`,
-          name: `${state} ${i}`,
-          coordinates: offsetCoordinates,
-        }
-
-        result.push(marker)
+      const marker: MapMarker = {
+        id: `${state}`,
+        name: `${state}`,
+        coordinates: [coordinates[0], coordinates[1]],
```

Comment:
Looks like the same thing

```suggestion
        id: state,
        name: state,
        coordinates,
```